.storybook/main.js

```diff
@@ -88,6 +88,19 @@ module.exports = {
       use: getStyleLoaders({ modules: true }),
     });
 
+    config.module.rules.unshift({
+      test: /\.svg$/,
+      use: [
+        {
+          loader: '@svgr/webpack',
+          options: {
+            svgo: false,
+          }
+        },
+        'url-loader'
+      ]
+    })
```

Comment:
missing `,;` eslint works fine?

order here ok? i thought url comes first. also url-loader or file-loader? i thought we go with file since we want to re-import the same files in different places

Reply:
it may not process hidden directories, will check

I think the order is alright - it works at least(?)

Doesn't really matter if we use file-loader or url-loader. This config only applies to storybook webpack and both loaders produce a URI

Reply:
ok here, might be not ok in rollup, might need file loader there

Reply:
Would prevent using like file-loader for rollup as long as possible. Will make it more difficult to setup apps to use our packages since they would need to serve the package's static files too.

Reply:
url loader would blow up the bundle size a lot. let's see if we need it but there might be no other way